addons/website/static/src/js/set_view_track.js

```diff
@@ -0,0 +1,74 @@
+odoo.define('website.set_view_track', function (require) {
+"use strict";
+
+var CustomizeMenu = require('website.customizeMenu');
+var rpc = require('web.rpc');
+var Widget = require('web.Widget');
+
+var TrackPage = Widget.extend({
+    template: 'website.track_page',
+    xmlDependencies: ['/website/static/src/xml/track_page.xml'],
+    events: {
+        'change #switch-track-page': '_onTrackChange',
+    },
+    track: null,
+    start: function () {
+        var self = this;
+        this.$input = this.$('#switch-track-page');
+        this._is_tracked().then(function (data) {
+            if (data[0]['track']) {
+                self.track = true;
+                self.$input.attr('checked', 'checked');
+            } else {
+                self.track = false;
+            }
+        });
+    },
+    _is_tracked: function (val) {
```

Comment:
- If it is a private method, it should be in a private section
- camelCase for javascript